trackpy/feature.py

```diff
@@ -118,12 +119,21 @@ def refine(raw_image, image, radius, coords, separation=0, max_iterations=10,
         estimated position
     max_iterations : integer
         max number of loops to refine the center of mass, default 10
+    engine : {'python', 'numba'}
```

Comment:
The `engine='auto'` code is in `locate()`, not here. Suggest we move it here as part of #257.